### PR TITLE
Feature info popup fix

### PR DESCRIPTION
**Pull request by matthewma7**
There was an issue for the feature info popup for timeseries layer.
And also, info for hidden layer was also shown.

This PR will fix popup for timeseries layer and make sure all hidden stuff won't show in the popup.

### CODE REVIEW

File: web_external/js/views/widgets/WMSFeatureInfoWidget.js [hunk 1]
```diff
@@ -25,24 +25,26 @@ minerva.views.WmsFeatureInfoWidget = minerva.View.extend({
         function getActiveGeojsonLayers() {
             var geojsonLayers = [];
             _.chain(that.parentView.collection.models)
-                .filter(function (set) { return set.get('displayed') && set.getDatasetType() === 'geojson'; })
+                .filter(function (set) { return set.get('displayed') && set.get('visible') && (set.getDatasetType() !== 'geojson' || set.getDatasetType() !== 'geojson-timeseries') })
```
Comment (manthey) [inline web_external/js/views/widgets/WMSFeatureInfoWidget.js]: `set.getDatasetType() !== 'geojson' || set.getDatasetType() !== 'geojson-timeseries'` will always be true.  I think you want `set.getDatasetType() === 'geojson' || set.getDatasetType() === 'geojson-timeseries'`

File: web_external/js/geojsonUtil.js [hunk 2]
```diff
@@ -114,7 +114,7 @@ minerva.geojson.normalize = function normalize(geojson) {  // eslint-disable-lin
                     normalized = $.extend({series: []}, norm);
                     normalized.summary = {};
                 }
-                var label = '' + (entry.label || entry.time || ('Frame ' + (normalized.series.length + 1)));
+                var label = '' + (entry.label || entry.time?moment(entry.time).format('L LTS'):null || ('Frame ' + (normalized.series.length + 1)));
```
Comment (manthey) [inline web_external/js/geojsonUtil.js]: Don't you want to use `entry.label` if it is set?  This functionally groups as `(entry.label || entry.time)?moment(entry.time).format('L LTS'):null || ('Frame ' + (normalized.series.length + 1))`, whereas I think you want `entry.label || (entry.time?moment(entry.time).format('L LTS'):null) || ('Frame ' + (normalized.series.length + 1))`

Comment (matthewma7) [inline web_external/js/geojsonUtil.js]: Totally. I should re-learn the basics again. 😅

File: web_external/js/views/widgets/WMSFeatureInfoWidget.js (hunk 1 above)
Comment (matthewma7) [inline web_external/js/views/widgets/WMSFeatureInfoWidget.js]: Totally.

File: web_external/js/views/widgets/WMSFeatureInfoWidget.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ minerva.views.WmsFeatureInfoWidget = minerva.View.extend({
 
         function getActiveWmsLayers() {
             return _.chain(that.parentView.collection.models)
-                .filter(function (set) { return set.get('displayed') && set.getDatasetType() !== 'geojson'; })
+                .filter(function (set) { return set.get('displayed') && set.get('visible') && set.getDatasetType() !== 'geojson' && set.getDatasetType() !== 'geojson-timeseries' })
```
Comment (manthey) [inline web_external/js/views/widgets/WMSFeatureInfoWidget.js]: This (and two other places below) throws a linting test for a missing semicolon.

Comment (matthewma7) [inline web_external/js/views/widgets/WMSFeatureInfoWidget.js]: Fixed. Thanks.

**Pull request merged**